tuning/config/configs.py

```diff
@@ -61,6 +61,10 @@ class ModelArguments:
                 tokenizer classes."
         },
     )
+    multimodal: bool = field(
```

Comment:
Is there a better way to tell if a user is loading a vision model and to then load the processor and load the model with `AutoModelForVision2Seq`? Should update this to vision specific flag